graphqa/util.py

```diff
@@ -94,7 +94,7 @@ def pretokenize_english(text):
     text = pretokenize_general(text)
     #for p in ENGLISH_PUNCTUATION:
     #    text = text.replace(p, f" {p} ")
-    table = string.maketrans({p: f" {p} " for p in ENGLISH_PUNCTUATION})
+    table = str.maketrans({p: f" {p} " for p in ENGLISH_PUNCTUATION})
```

Comment:
`str.maketrans` can be done in Global level

Reply:
Now put str.maketrans at the top of the script. Also created test scripts to test the related fucntions.